packages/block-library/src/button/style.scss

```
@@ -10,8 +10,8 @@ $blocks-button__height: 56px;
 	box-shadow: none;
 	cursor: pointer;
 	display: inline-block;
-	font-size: $big-font-size;
-	padding: 12px 24px;
+	font-size: 1.125em;
+	padding: 0.6em 1.1em;
```

This would be closer to the original value, wouldn't it?
```suggestion
	padding: 0.667em 1.333em;
```
...or if you only go to one decimal place:
```suggestion
	padding: 0.6em 1.3em;
```

---

Thank you for catching that! <3